packages/@okta/vuepress-site/docs/reference/architecture-center/directory-coexistence/lab-tutorial-3-generic-database/index.md

```diff
@@ -0,0 +1,316 @@
+   ```
+
+1. Run the okta-reference-coexistence-db-example/migrate-users.sh script.
+
+   ```bash
+   ./migrate-users.sh
+   ```
+
+This script uses a database query to the database server to get the user profiles, and the [Okta Users API](/docs/reference/api/users/) to import those users into Okta. For this example, three users are defined in `okta-reference-coexistence-db-example/blob/main/sql/03-users.sql`.
+
+The script imports the two users whose passwords are hashed with a hash that Okta supports: `Admin Istrator` and `User One`. The first time the users sign in, their passwords are rehashed and the hashed passwords are git-bstored in Okta Universal Directory using the `POST https://${yourOktaDomain}/api/v1/user`s` Okta API request.
+
+There is one more user in the generic database whose password is hashed with a hash that Okta does not support: **User2 Two**. This user is also imported into Universal Directory, but their password is not saved. In order to save a rehashed password the first time they sign in, an additional step is required. This is covered in [Account for hash types Okta does not support](#account-for-hash-types-okta-does-not-support).
+
+> **Note**: If you are using Windows, use `git-bash` to open a git-bash command window and then run `./migrate-users.sh` in the this window. Type `exit` to close it. The remainder of the example is performed in the original command window, not the git-bash command window.
+
+### Check the users have imported correctly
+
+Check the three imported users appear in your Okta Universal Directory.
+
+1. In a browser window, sign in to `https://${DOMAIN_NAME}-admin.okta.com`. If you do not know your `${OKTA_DOMAIN}`, see [Values and Variables](/docs/reference/architecture-center/directory-coexistence/lab-prerequisites/#values-and-variables).
+1. Choose **Directory > People**. You will see three users in addition to the users created in [Migrate users from an on-premises LDAP directory to Universal Directory](/docs/reference/architecture-center/directory-coexistence/lab-tutorial-2-ldap-server): **Admin Istrator**, **User One**, and **User2 Two**.
+
+   <div class="full border">
+
+   ![Okta people directory with three users listed](/img/architecture/directory-coexistence/db-to-okta-auth-migration-user1-user2-admin-createdbyscript.png)
+
+   </div>
+
+   > **Note:** If you do not see the three users, refresh the screen.
+
+   In the script, the user is imported and activated. [Activating a user account](https://help.okta.com/okta_help.htm?type=oie&id=ext_activate_user) sends an email to the user, informing them that their account is active and instructing them how to set up their account in your Okta org.
+
+1. Select User One to see the applications they are assigned to:
+
+   <div class="three-quarter border">
+
+   ![User 1 details, showing they are assigned to the okta-reference-coexistence-db-example application](/img/architecture/directory-coexistence/db-to-okta-user1-details.png)
```

Comment:
Again, I think this image is unnecessary, but will defer to other/team/yourself

Reply:
deleted